packages/react-ui/src/components/molecules/Alert.js

```diff
@@ -1,106 +1,155 @@
-import React, { useState } from 'react';
+import React, { forwardRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { Alert as MuiAlert, AlertTitle, Fade, styled } from '@mui/material';
 import Typography from '../atoms/Typography';
 import { ICON_SIZE_MEDIUM } from '../../theme/themeConstants';
 
 const StyledAlert = styled(MuiAlert, {
-  shouldForwardProp: (prop) => !['isNeutral', 'content', 'hasCloseButton'].includes(prop)
-})(({ isNeutral, content, hasCloseButton, theme }) => ({
-  paddingLeft: theme.spacing(1.5),
-  paddingRight: hasCloseButton ? theme.spacing(2) : theme.spacing(1.5),
-  display: 'grid',
+  shouldForwardProp: (prop) =>
+    ![
+      'isNeutral',
+      'content',
+      'hasCloseButton',
+      'hasAction',
+      'hasTitle',
+      'isSticky'
+    ].includes(prop)
+})(({ isNeutral, content, hasCloseButton, hasAction, hasTitle, isSticky, theme }) => ({
   columnGap: theme.spacing(1),
-  gridTemplateAreas:
-    content === 'inline' || hasCloseButton
-      ? `"icon message actions"`
-      : `
+  minHeight: theme.spacing(6),
+  padding: theme.spacing(1.5),
+
+  ...(hasAction && {
+    display: 'grid',
+    gridTemplateAreas:
+      content === 'inline' || hasCloseButton
+        ? `"icon message actions"`
+        : `
     "icon message"
     "icon actions"
-  `,
-  gridTemplateColumns: hasCloseButton
-    ? `${ICON_SIZE_MEDIUM} 1fr ${theme.spacing(2)}`
-    : `${ICON_SIZE_MEDIUM}`,
-  ...(isNeutral
-    ? {
-        backgroundColor: theme.palette.background.default,
-        color: theme.palette.text.primary
-      }
-    : {}),
+    `,
+    gridTemplateColumns: hasCloseButton
+      ? `${ICON_SIZE_MEDIUM} 1fr ${theme.spacing(3)}`
+      : `${ICON_SIZE_MEDIUM}`
+  }),
+  ...(isNeutral && {
+    backgroundColor: theme.palette.default.background,
+    color: theme.palette.text.primary
+  }),
+  ...(isSticky && {
+    borderRadius: 0
```

Comment:
It's a bit confusing that the sticky property changes the layout of the alert. Maybe we should introduce additional prop for the borderRadius change?

Reply:
I think sticky is term from design system, so it's proper term that abstracts implementation detail - border in this case.

https://www.figma.com/file/nmaoLeo69xBJCHm9nc6lEV/C4R-Components?type=design&node-id=9586-188422&mode=design&t=yujtHTpyoyJwKeT6-0

Reply:
Yes, exactly. Is an abstraction affecting just a border property in this case, but it could be another different think or a set of properties, so I would keep it as `sticky`.